lib/classifier-reborn/backends/bayes_memory_backend.rb

```diff
@@ -0,0 +1,72 @@
+# Author: Sawood Alam <@ibnesayeed>
```

Comment:
We don't have any author comments in our code – your authorship is present in the Git history and here on GitHub. 😄